python/src/main/python/drivers/runspv.py

```diff
@@ -70,25 +70,62 @@ def print(s):
     orig_print(s, flush=True)
 
 
-def subprocess_helper(cmd: List[str], check=False, timeout=None, stdout=None):
+def convert_and_print_stdout_stderr_returncode(result, verbose=False):
```

Comment:
Let's try to use argument types throughout the Python worker code as much as possible.

Reply:
I agree. I actually avoided it here because "result" is a mixture, but I can use Union